mainframer.sh

```diff
@@ -40,32 +61,21 @@ if [ -z "$BUILD_COMMAND" ]; then
 	exit 1
 fi
 
-pushd "$PROJECT_DIR"
+# Read exclude rules.
+LOCAL_EXCLUDE="$(readExcludesFile .mainframerignorelocal)"
```

Comment:
See #54. I'd vote for current names at the moment and then if we decide to move configs into a separate folder -> rename this files, names are not the point of this PR.